Apps/APN/CHANGELOG.md

```diff
@@ -0,0 +1,146 @@
+## [1.8.2](https://github.com/customerio/amiapp-reactnative/compare/1.8.1...1.8.2) (2023-03-28)
```

Comment:
This file looks like it was copied from the Ami app. I would delete it from this PR.

Reply:
Ah yes.